embedded-kafka/src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala

```diff
@@ -236,6 +236,7 @@ sealed trait EmbeddedKafkaSupport {
     props.put("group.id", s"embedded-kafka-spec")
     props.put("bootstrap.servers", s"localhost:${config.kafkaPort}")
     props.put("auto.offset.reset", "earliest")
+    props.put("enable.auto.commit", "false")
```

Comment:
I would feel better if this was a configuration value with a default `true` in order not to break backwards compatibility.